<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dnuevo</class>
 <widget class="QDialog" name="Dnuevo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="rb_int">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Int</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_char">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Char</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_nuevoArchivo">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>270</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Terminar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Archivo Nuevo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_agregarCampo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Campo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitud del campo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del campo</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbx_llave">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Campo llave</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del archivo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nameCampo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sp_longitud">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>48</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nameArchivo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
